packages/router/src/typed-routes/route-map.ts

```diff
@@ -17,16 +13,30 @@ export interface RouteRecordInfo<
   // TODO: could probably be inferred from the Params
   ParamsRaw extends RouteParamsRawGeneric = RouteParamsRawGeneric,
   Params extends RouteParamsGeneric = RouteParamsGeneric,
-  Meta extends RouteMeta = RouteMeta,
```

Comment:
I removed this type param because it was never used and I figured it's safer to add it back later if needed than leaving an unused type param that can _eventually_ be used. I marked this as a fix because the _leftover_ type param should have never been released in the first place